README.md

```diff
@@ -21,15 +21,16 @@ Optionally set `ROLLBAR_USERNAME` environment variable, usernames can be found a
 
 ### Inputs
 
-| Input          | Required  | Default      | Description                                      |
-| -------------- | --------- | ------------ | ------------------------------------------------ |
-| `environment`  | `true`    |              | The environment where the deploy is being done.  |
-| `version`      | `true`    |              | The version being deployed.                      |
-| `status`       | `false`   | `succeeded`  | The status of the deploy.                        |
-| `source_maps`  | `false`   |              | JS source map files.                             |
-| `minified_urls`| `false`   |              | Minified URLs linked to source maps above        |
+| Input           | Required  | Default      | Description                                      |
+| --------------- | --------- | ------------ | ------------------------------------------------ |
+| `environment`   | `true`    |              | The environment where the deploy is being done.  |
+| `version`       | `true`    |              | The version being deployed.                      |
+| `status`        | `false`   | `succeeded`  | The status of the deploy.                        |
+| `source_maps`   | `false`   |              | JS source map files.                             |
+| `minified_urls` | `false`   |              | Minified URLs linked to source maps above        |
+| `local_username`| `false`   |              | Username of the deploying user. Alternative to setting ROLLBAR_USERNAME |
```

Comment:
Description can be "Github username of the deploying user" since this action is specific to Github.

Reply:
Thanks for the response @mrunalk.

I think this parameter could in theory be used to specify non-GitHub usernames, right? I left it generic for that reason. But I agree that the most common use case in the context of this action would be to send GitHub usernames.

Reply:
Looking over the docs again, I realized there was an existing note in the `README` about usernames - I adjusted it in ebc3fcdbbbb9821f42988fa8b9de67da1147a5f9.